src/rpdk/core/data/schema/provider.definition.schema.v1.json

```diff
@@ -71,6 +71,11 @@
                 }
             }
         },
+        "taggable": {
+            "$comment": "A boolean flag indicating whether this resource type supports updatable tagging.",
+            "type": "boolean",
+            "default": "true"
+        },
```

Comment:
This isn't the same place as the [`resource-schema` PR](https://github.com/aws-cloudformation/aws-cloudformation-resource-schema/pull/105/files?file-filters%5B%5D=.json), and this is already out of sync otherwise anyway

Let's not manually maintain duplicates, I'll set up something to automatically sync this from [`aws-cloudformation-resource-schema`](https://github.com/aws-cloudformation/aws-cloudformation-resource-schema/blob/master/src/main/resources/schema/provider.definition.schema.v1.json) later